mongoengine/document.py

```diff
@@ -434,7 +434,7 @@ def delete(self):
         try:
             self._qs.filter(**self._object_key).delete(_from_doc_delete=True)
         except pymongo.errors.OperationFailure, err:
-            message = u'Could not delete document (%s)' % err.message
+            message = u'Could not delete document (%s)' % str(message)
```

Comment:
ticket is mainly for this guy. we have some deletions in mongoengine and that will potentially cause a machine to go into error state and not recover

Reply:
I think this should be `str(err)`. I don't see a variable named `message` in scope. :-)

Reply:
(knocks head against table)